### DISCUSSION TITLE
Why the network connection is being made

### DISCUSSION THREAD

**Discussion by siddharthapradhan1** [Open-Systems-Pharmacology/Forum#796] [Questions & Problems] [answered]
Connection to 151.101.40.133 on tcp/443 (raw.githubusercontent.com). 

After the installation of PK-Sim, we found this network connection is being made. 
Why the network connection is being made and what is the reason behind it? 
Is there any data being transfer from user machine to PK-Sim Server? 
Is there any cloud service play with PK-Sim?

**Comment by Yuri05** [comment, marked as answer]
>Why the network connection is being made and what is the reason behind it?

PK-Sim (and MoBi) checks during program start if a newer released version is available on GitHub and informs the user if this is the case.

>Is there any data being transfer from user machine to PK-Sim Server?

no

>Is there any cloud service play with PK-Sim?

no

**Comment by siddharthapradhan1** [comment]
Hi Juri,

Thanks much for the information.
If we need more information based on our Gilead Security Leader comment, we shall connect you.

Regards,
Siddhartha

**Comment by StephanSchaller** [comment]
Dear Siddhartha,

what do you mean by "Gilead Security Leader comment"?

Thank you,
Stephan

**Comment by siddharthapradhan1** [comment]
Hi Juri,

Before installing any software within Gilead organization, we do Software Security Assessment to find out if there is any loophole or any external network connections are being made.

Based on your comment, we will prepare an assessment report for PK-Sim and send to our Security Leader for their review and approval.

So that i mentioned this.

Let me know in case any queries.

Regards,
Siddhartha
________________________________
